docs/websites/ror/host-multiple-rails-apps-with-nginx-puma-postgres-and-rbenv-ubuntu-14-04-trusty-tahr.md

```diff
@@ -15,96 +15,99 @@ contributor:
+This guide provides step-by-step instructions for hosting multiple Rails apps on Ubuntu 14.04 (Trusty Tahr) with Linode.
 
-Before you begin ensure that you have followed the [Getting Started](https://www.linode.com/docs/getting-started) and [Securing Your Server](https://www.linode.com/docs/security/securing-your-server/) guides.
+Before you begin, ensure that you have followed the [Getting Started](https://www.linode.com/docs/getting-started) and [Securing Your Server](https://www.linode.com/docs/security/securing-your-server/) guides.
 
-This guide assumes that you have created, and are logged in as, a non-root user with sudo privileges. This guide uses the name rick, but update your commands appropriately.
+This guide assumes that you have created and are logged in as a non-root user with sudo privileges. This guide uses the name rick, but you can use whatever name you choose and alter your commands accordingly.
 
 ##Configuring your first Rails app
 
 ### Install Postgres
 
-1.  Update the system and install essentials
+1.  Update the system and install essentials by entering: 
 
         sudo apt-get update
         sudo apt-get upgrade --show-upgraded
         sudo apt-get install -y build-essential libssl-dev libreadline-dev zlib1g-dev git postgresql postgresql-contrib libpq-dev nodejs nginx
 
-2.  Setup a postgres user with the same name as the current user
+2.  Setup a postgres user with the same name as the current user:
 
         sudo -u postgres createuser --superuser $USER
+
+    Log into postgres and set the password for your user, if you wish:
+
         sudo -u postgres psql
-        \password $USER
+        \password username
 
-As this guide assumes you will be using a local Postgres database you may wish to leave the password blank
+    This guide assumes you will be using a local Postgres database; thus, you may wish to leave the password blank.
 
-3.  To exit Postgres
+3.  To exit Postgres, type: 
 
         \q
 
 
 ### Install rbenv
 
-1.  Install rbenv as user
+1.  To install rbenv as user, type:
 
         git clone https://github.com/sstephenson/rbenv.git ~/.rbenv
         echo 'export PATH="$HOME/.rbenv/bin:$PATH"' >> ~/.bashrc
         echo 'eval "$(rbenv init -)"' >> ~/.bashrc
 
-2.  Restart shell (log out and back in) and check that rbenv is installed
+2.  Now, restart the shell (log out and back in) and check that rbenv is installed by entering: 
 
         type rbenv
 
-3.  Install ruby-build as an rbenv plugin
+3.  To install ruby-build as an rbenv plugin, enter: 
 
         git clone https://github.com/sstephenson/ruby-build.git ~/.rbenv/plugins/ruby-build
 
-4.  Install rbenv-vars to manage environment variables
+4.  To install rbenv-vars to manage environment variables, enter:
 
         git clone https://github.com/sstephenson/rbenv-vars.git ~/.rbenv/plugins/rbenv-vars
 
-5.  Install latest version of Ruby
+5.  To install the latest version of Ruby, enter: 
```

Comment:
Saying "the latest version" and then specifically calling out a version does not make this guide future proof. This should say "latest as of publication" and have a reference link on where to find the current latest.